src/setops.jl

```diff
@@ -2,13 +2,13 @@
 # Projection operations
 # ========================================
 
-function _Projection(X::AbstractLazyReachSet, st_vars)
-    Projection(X, st_vars)
+function _project_oa(X::AbstractLazyReachSet, vars)
+    return Project(X, vars)
```

Comment:
```suggestion
    return project(X, vars)
```
Not sure if you wanted to have a concrete set here. At least you removed the comment "lazy set".

Reply:
i think we can keep the lazy set here. and let the network decide how to handle it